go.mod

```diff
@@ -6,7 +6,7 @@ toolchain go1.23.7
 
 require (
 	github.com/google/go-cmp v0.7.0
-	github.com/hashicorp/terraform-plugin-go v0.29.0-alpha.1.0.20250709165734-a8477a15f806
+	github.com/hashicorp/terraform-plugin-go v0.29.0-alpha.1.0.20250717120746-d48df0b0147b
```

Comment:
TODO: Update this before merging, once the plugin-go PR has been merged to `main`